ui/src/app/shared/components/checkbox-filter/checkbox-filter.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
```

Comment:
`checkbox-filter` introduces a list of checkboxes with counts displayed for for each item.